cellprofiler/modules/namesandtypes.py

```diff
@@ -314,7 +314,7 @@ def create_settings(self):
                 doc=LOAD_AS_CHOICE_HELP_TEXT)
 
         self.volumetric = cps.Binary(
-            text="Volumetric",
+            text="Data is 3D",
```

Comment:
@bethac07 and @karhohs could chime in. Requires thinking about the context in which this is being asked.